processing/src/test/java/org/apache/druid/segment/UnnestColumnValueSelectorCursorTest.java

```diff
@@ -592,7 +590,25 @@ public void test_list_unnest_cursors_dimSelector()
         OUTPUT_NAME,
         IGNORE_SET
     );
-    unnestCursor.getColumnSelectorFactory().makeDimensionSelector(DefaultDimensionSpec.of(OUTPUT_NAME));
+    // should return a column value selector for this case
+    BaseSingleValueDimensionSelector unnestDimSelector = (BaseSingleValueDimensionSelector) unnestCursor.getColumnSelectorFactory()
+                                                                                                        .makeDimensionSelector(
+                                                                                                            DefaultDimensionSpec.of(
+                                                                                                                OUTPUT_NAME));
+    unnestDimSelector.inspectRuntimeShape(null);
+    int k = 0;
+    while (!unnestCursor.isDone()) {
+      if (k < 8) {
+        Assert.assertEquals(unnestDimSelector.getValue(), expectedResults.get(k).toString());
```

Comment:
I think you have expected and actual inverted.  The junit assert has expected come first.

Reply:
Thanks, actually in this file all were inverted. Good catch ! I'll update all of them